### PR TITLE
Add App Icon to Notification

### PR DESCRIPTION
**Pull request by williamleuschner**
This proposed pull request adds the capability for `notification-center` to use the WeeChat icon in it's notification. As a caveat, the `terminal-notifier` application states that this is an experimental feature, and, as it uses private APIs, is likely to break. However, it works well now, and it improves the overall user experience. Thank you in advance for your consideration.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please only do the actual changes, style changes should be a separate PR.

**Comment by williamleuschner** [comment]
Oh, whoops! I made the PEP8 changes out of habit and didn't even realize it would be a problem. I'll fix this later today and re-submit the pull request.

**Comment by sindresorhus** [comment]
No need to resubmit. Just update this PR.

https://github.com/RichardLitt/docs/blob/master/amending-a-commit-guide.md

**Comment by williamleuschner** [comment]
There, that should be good.

**Comment by sindresorhus** [comment]
Can you include the icon in this PR with the correct name? I want to ensure it doesn't go away at some point.

File: notification_center.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 SCRIPT_VERSION = '1.1.0'
 SCRIPT_LICENSE = 'MIT'
 SCRIPT_DESC = 'Pass highlights and private messages to the OS X 10.8+ Notification Center'
+WEECHAT_ICON = '.weechat/weechat.png'
```
Comment (sindresorhus) [inline notification_center.py]: Shouldn't this use `os.path.expanduser()` or something?

File: readme.md [hunk 2]
```diff
@@ -10,12 +10,14 @@
 ### Script center
 
 - Install [pync](https://github.com/SeTeM/pync): `pip install pync`
+- Download [the WeeChat icon](https://weechat.org/media/images/weechat_logo_small.png) to `$HOME/.weechat/weechat.png`
```
Comment (sindresorhus) [inline readme.md]:
```
`~/.weechat/weechat.png`
```

File: readme.md [hunk 3]
```diff
@@ -10,12 +10,14 @@
...
 - In WeeChat, type `/script` to open the script manager
 - Find `notification_center` and type `i` then `Enter` to install
 
 ### Manually
 
 - Install [pync](https://github.com/SeTeM/pync): `pip install pync`
+- Download [the WeeChat icon](https://weechat.org/media/images/weechat_logo_small.png) to `$HOME/.weechat/weechat.png`
```
Comment (sindresorhus) [inline readme.md]:
```
`~/.weechat/weechat.png`
```

**Comment by sindresorhus** [comment]
I don't have the ability to test this as I no longer have WeeChat installed. I trust you've tested this?

**Comment by williamleuschner** [comment]
1. Yes, I see now that that is a mistake. 
2. I'll add that
3. Yes, I've tested it.

**Comment by williamleuschner** [comment]
There, I've made the requested changes.

**Comment by sindresorhus** [comment]
That's one ugly logo :p

**Comment by sindresorhus** [comment]
Thanks, looks good. Submitted to the WeeChat script center thing.

**Pull request closed without merging**